src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.lerp(a1, a2, amt) * colorModeA + 0.5 | 0;
```

Comment:
I tested in the browser console, `0.5 + 0.5 | 0` evaluates to `1`; `0.5 + (0.5 | 0)` evaluates to `0.5`.